rllib/core/examples/simple_ppo_rl_module.py

```diff
@@ -278,7 +278,7 @@ def input_specs_train(self) -> ModelSpec:
                 "obs": self.encoder.input_specs()
                 if self.encoder
                 else self.pi.input_specs(),
-                "action": TorchTensorSpec("b, da", da=action_dim),
+                "actions": TorchTensorSpec("b, da", da=action_dim),
```

Comment:
needed to do this to make compatible with sample batch

Reply:
Good catch. Can we use the `SampleBatch.ACTIONS` constant instead (and everywhere else in this file?).

Reply:
ill do that in a separate pr